openadapt/config.py

```diff
@@ -110,6 +114,37 @@ def getenv_fallback(var_name):
     return rval
 
 
+def persist_env(var_name: str, val: str, env_file_path: str = ENV_FILE_PATH):
+    """
+    Persist environment variables by writing them to a file or
+    updating their values if they already exist in the file.
+
+    Args:
+      var_name: A string that represents the name of the environment
+          variable to persist or update.
+      val: The value to assign to the environment variable.
+    """
+    if not os.path.exists(env_file_path):
+        with open(env_file_path, "w") as f:
+            f.write(f"{var_name}={val}")
+    else:
+        # find and replace
+        with open(".env", "r") as f:
+            lines = f.readlines()
+        for i, line in enumerate(lines):
+            if line.startswith(f"{var_name}="):
+                lines[i] = f"{var_name}={val}\n"
+                break
```

Comment:
```suggestion
        lines = open(ENV_FILE_PATH, "r").readlines()
        lines = [
            line
            for line in lines
            if not line.startswith(f"{var_name}=")
            else f"{var_name}={val}\n"
        ]
```

Reply:
while this logic does update the variable, it doesn't append at the end if the variable is not found. I think my for/else works better